devices/tuya.js

```diff
@@ -43,6 +43,45 @@ const tzLocal = {
             }
         },
     },
+    indicate_switch: {
+        key: ['indicate_switch'],
+        convertSet: async (entity, key, value, meta) => {
+            const lookup = {'off': 0, 'switch': 1, 'position': 2};
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(lookup));
+            const indicate = lookup[value];
+            await entity.write('genOnOff', {tuyaBacklightMode: indicate});
+            return {state: {indicate_switch: value}};
+        },
+        convertGet: async (entity, key, meta) => {
+            await entity.read('genOnOff', ['tuyaBacklightMode']);
+        },
+    },
+    backlight: {
+        key: ['backlight'],
+        convertSet: async (entity, key, value, meta) => {
+            const backlight = value === 'ON' ? 1 : 0;
+            await entity.write('genOnOff', {tuyaBacklightSwitch: backlight});
+            return {state: {backlight}};
+        },
+        convertGet: async (entity, key, meta) => {
+            await entity.read('genOnOff', ['tuyaBacklightSwitch']);
+        },
+    },
+    power_on_behavior: {
+        key: ['tuya_power_on_behavior'],
```

Comment:
Please rename this to `power_on_behavior`

Reply:
This device also uses `moes_power_on_behavior` converter which also has `power_on_behavior` key. I have tested and if it is renamed key to `power_on_behavior` it doesn't work correctly, `moes_power_on_behavior` converter calls earlier, so that's why I added "tuya" prefix.

Reply:
What is the reason to use two `power_on_behavior` converters? If I look at `moes_power_on_behavior` is provides the same functionallity.

Reply:
`moes_power_on_behavior` uses `genOnOff` cluster. This converter, with prefixed key, uses `manuSpecificTuya_v3` cluster.
`moes_power_on_behavior` switches "power on bevahior" for the whole device and all channels, whereas this converter switches "power on behavior" for the each channel separately.

Reply:
I see, thanks for the explanation, I think it would be better to expose just one `power_on_behaviour` (the `manuSpecificTuya_v3` one)

Reply:
Done, https://github.com/Koenkk/zigbee-herdsman-converters/pull/4301/commits/e19b5d069e9474f6fdd56fdcf983791c91ed954f